cmd/glasskube/cmd/update.go

```diff
@@ -117,9 +119,54 @@ func completeInstalledPackageNames(
 	return
 }
 
+func completeUpgradablePackageVersions(
+	cmd *cobra.Command,
+	args []string,
+	toComplete string,
+) ([]string, cobra.ShellCompDirective) {
+
+	var dir cobra.ShellCompDirective
+	config, _, err := kubeconfig.New(config.Kubeconfig)
+	if err != nil {
+		dir = cobra.ShellCompDirectiveError
+		return nil, dir
+	}
+	client, err := client.New(config)
+	if err != nil {
+		dir = cobra.ShellCompDirectiveError
+		return nil, dir
+	}
+	if len(args) == 0 {
+		return nil, dir
+	}
+	packageName := args[0]
+	var packageIndex repo.PackageIndex
+	if err := repo.FetchPackageIndex("", packageName, &packageIndex); err != nil {
+		return nil, dir
+	}
+	if len(args) == 0 {
+		return nil, dir
+	}
+	var pkg v1alpha1.Package
+	if err := client.Packages().Get(cmd.Context(), packageName, &pkg); err != nil {
+		dir &= cobra.ShellCompDirectiveError
+		return nil, dir
+	}
+	versions := make([]string, 0, len(packageIndex.Versions))
+	for _, version := range packageIndex.Versions {
+		if toComplete == "" || strings.HasPrefix(version.Version, toComplete) {
+			if !semver.IsUpgradable(pkg.Spec.PackageInfo.Version, version.Version) {
```

Comment:
The negation here is incorrect. The current implementation suggests all versions that can NOT be upgraded to, but we obviously need the opposite :wink: